### PR TITLE
Debug mode menu step 1

### PR DESCRIPTION
**Pull request by ThibaudAV**
**Issue**

https://github.com/gravitee-io/issues/issues/6893
https://github.com/gravitee-io/issues/issues/6892

**Description**

Prepare for next PR to add menu for PS

- remove unused PS wrapper in angularjs
- change target to es6===es2015 

**Additional context**

---

📚&nbsp;&nbsp;View the storybook of this branch [here](https://612657caa8e859003a8a6430-ouwcrkrmuj.chromatic.com)

🚀 CI was able to deploy the build of this PR, so you can now try it directly [here](https://apimnightlywebui24386.blob.core.windows.net/debug-mode-menu-step-1/index.html)
_Notes_: The deployed app is linked to the management API of APIM master. (Same login and password as APIM master)

### CODE REVIEW

**Comment by ThibaudAV** [comment]
There is a weird thing. It works locally but it doesn't seem to work on the :
> CI was able to deploy the build of this PR, so you can now try it directly

But I don't know yet if it comes from this PR

File: gravitee-apim-console-webui/package.json [hunk 1]
```diff
@@ -62,6 +62,7 @@
     "md-steppers": "0.4.1",
     "messageformat": "2.3.0",
     "moment": "2.29.1",
+    "ng-annotate-patched": "^1.13.0",
```
Comment (gaetanmaisse) [inline gravitee-apim-console-webui/package.json]: It's a dev dependency no?

File: gravitee-apim-console-webui/src/management/management.module.ajs.ts [hunk 2]
```diff
@@ -547,19 +546,15 @@ require('diff/dist/diff.min.js');
...
-import NoDataToDisplay from 'highcharts/modules/no-data-to-display';
-import Map from 'highcharts/modules/map';
-
-HighchartsMore(Highcharts);
-SolidGauge(Highcharts);
-NoDataToDisplay(Highcharts);
-Map(Highcharts);
-
-import '@highcharts/map-collection/custom/world';
+const Highcharts = require('highcharts/highstock');
```
Comment (gaetanmaisse) [inline gravitee-apim-console-webui/src/management/management.module.ajs.ts]: Is should be `highcharts` and not `highcharts/highstock` no?

File: gravitee-apim-console-webui/src/management/management.module.ajs.ts [hunk 3]
```diff
@@ -547,19 +546,15 @@ require('diff/dist/diff.min.js');
...
+window.Highcharts = Highcharts;
+require('highcharts/modules/exporting')(Highcharts);
```
Comment (gaetanmaisse) [inline gravitee-apim-console-webui/src/management/management.module.ajs.ts:549]: Needed?

**Comment by gaetanmaisse** [comment]
:aa: It looks like the snackBar isn't super happy. It's certainly not related to this PR as it fails in Policy Studio and works in other screens. Maybe related to Angular Material stuff?

image

File: gravitee-apim-console-webui/package.json (hunk 1 above)
Comment (ThibaudAV) [reply gravitee-apim-console-webui/package.json]: Yes, it is. i fix it, ty

File: gravitee-apim-console-webui/src/management/management.module.ajs.ts (hunk 3 above)
Comment (ThibaudAV) [reply gravitee-apim-console-webui/src/management/management.module.ajs.ts:549]: It seems to me Yes it is. needed for es6 IMHO But highcharts was very complicated to make work in es6 🪦 . As it seems to work, I don't really want to modify it 🙈

File: gravitee-apim-console-webui/src/management/management.module.ajs.ts (hunk 2 above)
Comment (ThibaudAV) [reply gravitee-apim-console-webui/src/management/management.module.ajs.ts]: I don't know, we can try. I found this solution in an issue of highcharts about es6

Comment (ThibaudAV) [reply gravitee-apim-console-webui/src/management/management.module.ajs.ts]: :youhoo: it's work. 🚀

**Comment by ThibaudAV** [comment]
i've add two new commit to fix your message related subject,  it's easy so i add it in this PR ;)

**Review by gcusnieux — APPROVED** [review]

**Pull request merged**